cmd/pool/account.go

```diff
@@ -173,29 +207,37 @@ func newAccount(ctx *cli.Context) error {
 }
 
 func printAccountFees(client poolrpc.TraderClient, amt btcutil.Amount,
-	confTarget uint32) error {
+	satPerVByte uint64, confTarget uint32) error {
+
+	if confTarget != 0 {
```

Comment:
nit: If we used `if confTarget == 0 {` we could have the smaller code block indented and have the larger block below without the additional level of indentation.